Not able to generate javadoc due to errors in the javadoc code

The current version 0.0.2-SNAPSHOT has 5 errors in javadoc, javadoc cannot be generated.


```
15:36:45  > Task :javadoc FAILED
15:36:45  /jenkins/workspace/Adapters/IntelliJ/intellij-common-ui-test-library-build/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/utils/testExtension/ScreenshotAfterTestFailExtension.java:31: error: unknown tag: ExtendWith
15:36:45   *        @ExtendWith(ScreenshotAfterTestFailExtension.class)
15:36:45            ^
15:36:45  /jenkins/workspace/Adapters/IntelliJ/intellij-common-ui-test-library-build/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/UITestRunner.java:51: warning: no @return
15:36:45      public static RemoteRobot runIde(IdeaVersion ideaVersion, int port) {
15:36:45                                ^
15:36:45  /jenkins/workspace/Adapters/IntelliJ/intellij-common-ui-test-library-build/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/UITestRunner.java:77: warning: no @return
15:36:45      public static RemoteRobot runIde(IdeaVersion ideaVersion) {
15:36:45                                ^
15:36:45  /jenkins/workspace/Adapters/IntelliJ/intellij-common-ui-test-library-build/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/UITestRunner.java:117: warning: no @throws for java.lang.InterruptedException
15:36:45      public static RemoteRobot getRemoteRobotConnection(int port) throws InterruptedException {
15:36:45                                ^
15:36:45  /jenkins/workspace/Adapters/IntelliJ/intellij-common-ui-test-library-build/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/mainIdeWindow/toolWindowsPane/ToolWindowsPane.java:60: warning: no @param for label
15:36:45      public ComponentFixture stripeButton(String label) {
15:36:45                              ^
15:36:45  /jenkins/workspace/Adapters/IntelliJ/intellij-common-ui-test-library-build/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/mainIdeWindow/toolWindowsPane/ToolWindowsPane.java:118: warning: no @return
15:36:45      public boolean isAProjectFilePresent(String... path) {
15:36:45                     ^
15:36:45  1 error
15:36:45  5 warnings
```

--- a/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/UITestRunner.java
+++ b/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/UITestRunner.java
@@ -47,6 +47,7 @@ public class UITestRunner {
      *
      * @param ideaVersion version of the IntelliJ Idea to start
      * @param port        port number on which will the IntelliJ Idea be listening
+     * @return instance of the RemoteRobot
      */
     public static RemoteRobot runIde(IdeaVersion ideaVersion, int port) {
         UITestRunner.ideaVersion = ideaVersion;
@@ -73,6 +74,7 @@ public class UITestRunner {
      * Start the given version of IntelliJ Idea listening on the default port
      *
      * @param ideaVersion version of the IntelliJ Idea to start
+     * @return instance of the RemoteRobot
      */
     public static RemoteRobot runIde(IdeaVersion ideaVersion) {
         return runIde(ideaVersion, defaultPort);
@@ -113,6 +115,7 @@ public class UITestRunner {
      *
      * @param port port number
      * @return instance of the RemoteRobot
+     * @throws InterruptedException may be thrown in Thread.sleep()
      */
     public static RemoteRobot getRemoteRobotConnection(int port) throws InterruptedException {
         RemoteRobot remoteRobot = new RemoteRobot("http://127.0.0.1:" + port);

--- a/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/mainIdeWindow/toolWindowsPane/ToolWindowsPane.java
+++ b/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixtures/mainIdeWindow/toolWindowsPane/ToolWindowsPane.java
@@ -55,6 +55,7 @@ public class ToolWindowsPane extends CommonContainerFixture {
     /**
      * Create fixture for the Stripe button
      *
+     * @param label label text of the stripe button
      * @return fixture for the Stripe button
      */
     public ComponentFixture stripeButton(String label) {
@@ -113,7 +114,8 @@ public class ToolWindowsPane extends CommonContainerFixture {
     /**
      * Test is a file with given name on given path is available in the project tree
      *
-     * @param path        path to navigate
+     * @param path path to navigate to
+     * @return true if the given file exists on the given path in the project
      */
     public boolean isAProjectFilePresent(String... path) {
         try {

--- a/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/utils/testExtension/ScreenshotAfterTestFailExtension.java
+++ b/src/main/java/com/redhat/devtools/intellij/commonUiTestLibrary/utils/testExtension/ScreenshotAfterTestFailExtension.java
@@ -25,10 +25,10 @@ import static com.redhat.devtools.intellij.commonUiTestLibrary.utils.screenshot.
 
 /**
  * ScreenshotAfterTestFailExtension takes screenshot immediately after test has failed
- * and perform a clean up to ensure no dialog or windows is opened
+ * and perform a clean up to ensure no dialog or windows is opened<br>
  *
- * USAGE: Add the following annotation before every class with JUnit tests
- *        @ExtendWith(ScreenshotAfterTestFailExtension.class)
+ * <b>USAGE:</b> Add the following annotation before every class with JUnit tests:
+ * {@code @ExtendWith(ScreenshotAfterTestFailExtension.class)}<br>
  *
  * @author zcervink@redhat.com
  *

